recipes/exprtk/all/test_package/CMakeLists.txt

```diff
@@ -1,17 +1,15 @@
-cmake_minimum_required(VERSION 3.1)
+cmake_minimum_required(VERSION 3.8)
 
-project(test_package)
+project(test_package LANGUAGES CXX)
 
-include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
-
-conan_basic_setup()
+find_package(exprtk REQUIRED CONFIG)
 
 add_executable(${PROJECT_NAME} test_package.cpp)
 
-target_link_libraries(${PROJECT_NAME} ${CONAN_LIBS})
+target_link_libraries(${PROJECT_NAME} PRIVATE exprtk::exprtk)
 
 if (MSVC)
-  target_compile_options(${PROJECT_NAME} PRIVATE "/bigobj")
+   # Compile will fail without this flag
+   # TODO should this be something added into conan recipe?
+   target_compile_options(${PROJECT_NAME} PRIVATE "/bigobj")
```

Comment:
Generally not, it's possible to use a smaller peice of the lirbary and then you would not want to pay the performance penalty ... doing a quick search it's never exposed

Reply:
/bigobj is required for MSVC compiler to not die when building.
It may always be required to use this particular library, I'm not sure.
I'll leave it out of the recipe, but in the test_package (otherwise it won't build)